onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -549,3 +551,39 @@ def clone_model_with_shape_infer(model):
     else:
         cloned_model = save_and_reload_model(model)
     return cloned_model
+
+
+def tensor_proto_to_array(initializer):
+    if initializer.data_type == onnx_proto.TensorProto.FLOAT:
+        weights = onnx.numpy_helper.to_array(initializer)
+    else:
+        raise ValueError(
+            "Only float type quantization is supported. Weights {} is {}. ".format(
```

Comment:
C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
Raw output